core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1040,17 +1044,23 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
   test("reserve/release unroll memory") {
     store = makeBlockManager(12000)
     val memoryStore = store.memoryStore
+    val dummyBlock = TestBlockId("")
```

Comment:
Since these variables are only used within `reserveUnrollMemoryForThisTask` you could define them inside of that method.